### DISCUSSION TITLE
Who started the initiative to start beemod?

### DISCUSSION THREAD

**Discussion by CN1o2** [BEEmod/BEE2-items#3897] [General]
Just for curiosity

**Comment by vrad-exe** [comment]
It was originally created by BenVlodgi, but he doesn't really work on the mod anymore. Here was his explanation of BEE's history from Discord:

> Originally it was Ben’s Extended Editor Mod (BEE Mod).
Then August made the editor models for me, so I added his name to the Title. When making mods for free, credit is the main payment (plus satisfied users :relaxed:)
At some point I did re-arrange our names to be in alphabetical order on the title..

> Then I was making a new program that could build editor palettes.
Carl had separately made his own mod, Stylechanger. Which would change the style of the chamber. He also made it compatible with BEEMod.
I told Carl about the new BEE2.1 I had made with a pallet designer. It was nearly finished, when we decided to combine forces and add style chamber features to BEE2. His style chamber code was written in Delphi. We (mostly him) rewrote it in C#. We ended up releasing BEE2.2 together. Ben and Carl’s Extended Editor.
That version of BEE2 always had bugs that were so hard to fix due to all the complicated way we integrated style stuff. There was also a user generated package that was super buggy and we shouldn’t have shipped with the Mod

> Teamspen had made fixes for the BEE2.2.5 and published them separately, which was awesome.

> I had begun working on a rewrite of BEE2.2 visual code to be BEE2.3. It was a bit better, but wouldn’t run at all on Mac. I eventually dropped it.

> Then Spencer and I were talking. And the idea of rewriting the BEE2 interface began. Also rewriting the package structure to support versioning and common assets. He came up with the idea of using a cross platform language so it would work on more than just windows.

> While I was involved in the initial planning for the BEE2.4, it really was Spencer doing all the work. Maybe it was entertained to call it Ben and Spencer’s Extended editor? I don’t think we ever called it that. 
At the time I don’t think Teamspen was publicly using his real name yet. 
Better Extended Editor felt like a good title change. Since there were many volunteers getting involved in various ways, and I wasn’t contributing anymore, it felt weird to have my name take so much credit.
